auth-web/src/views/auth/home/IncorpOrRegisterView.vue

```diff
@@ -14,41 +14,116 @@
+                      <v-expansion-panel-header class="incorp-expansion-header font-weight-bold">
+                        <template v-slot:actions>
+                          <v-icon color="primary">
+                            $expand
+                          </v-icon>
+                        </template>
+                        <span class="tooltip-text" v-on="on">B.C. Based Company</span>
+                      </v-expansion-panel-header>
+                      <v-expansion-panel-content>
+                        <span id="expPnlContent2" class="list-item-text py-3">
+                          Office addresses, director names and addresses, share structure and articles.</span>
+                      </v-expansion-panel-content>
+                    </v-expansion-panel>
                   </template>
-                  {{ item.text }}
-                </v-expansion-panel-header>
-                <v-expansion-panel-content>
-                    <v-list-item-content>
-                      <v-list-item class="list-item" v-for="(subItem, subIndex) in item.items" :key="subIndex">
-                        <v-list-item-content>
-                          <v-list-item-subtitle class="list-item-text py-3">
-                            {{subItem.text}}
-                          </v-list-item-subtitle>
-                        </v-list-item-content>
-                      </v-list-item>
-                    </v-list-item-content>
-                </v-expansion-panel-content>
-              </v-expansion-panel>
-            </v-expansion-panels>
+                  <v-card class="tooltip-content">
+                    <div>
+                      <h3 class="mb-3">B.C. Based Company</h3>
+                      <span>You can incorporate the following B.C. based company types: Limited Company, Unlimited
+                        Liability Company, Benefit Company, and Community Contribution Company.</span>
+                    </div>
+                  </v-card>
+                </v-tooltip>
+                <v-expansion-panel class="incorp-expansion-panels">
+                  <v-expansion-panel-header class="incorp-expansion-header font-weight-bold">
+                    <template v-slot:actions>
+                      <v-icon color="primary">
+                        $expand
+                      </v-icon>
+                    </template>
+                    Cooperative Association
+                  </v-expansion-panel-header>
+                  <v-expansion-panel-content>
+                    <span id="expPnlContent3" class="list-item-text py-3">
+                      Office addresses, director names and addresses, rules of the association
+                      and memorandum.</span>
+                  </v-expansion-panel-content>
+                </v-expansion-panel>
+              </v-expansion-panels>
+            </div>
+
+            <!-- For BEN only as feature flag 'EnableBcCccUlc' enabled. Remove section when FF removed -->
```

Comment:
I'm OK with this code duplication as long as we have a future ticket to remove it -- probably the same ticket to remove the FF after we deploy the new incorporation types to Prod (may need a new ticket under epic 14108).